src/relayserver/RelayServer.ts

```diff
@@ -65,21 +65,7 @@ function spam (...args: any): void {
 class StateError extends Error {
 }
 
-export interface CreateTransactionDetails extends GsnTransactionDetails {
-  // todo: gasLimit defined as "gas"
-  gasLimit: PrefixedHexString
-  gasPrice: PrefixedHexString
-  // todo: encodedFunction defined as "data"
-  approvalData: PrefixedHexString
-  signature: PrefixedHexString
-  senderNonce: IntString
-  relayMaxNonce: IntString
-  baseRelayFee: IntString
-  pctRelayFee: IntString
-  relayHubAddress: Address
-  paymaster: Address
-  forwarder: Address
-}
+export type CreateTransactionDetails = TmpRelayTransactionJsonRequest
```

Comment:
exactly

Reply:
This should be removed when removing tmpRelayTransactionJsonRequest